circle.yml

```diff
@@ -14,6 +14,14 @@ dependencies:
   cache_directories:
     - "~/.cabal"
 
+  override:
+    - mkdir -p /home/ubuntu/.go_workspace/src/github.com/Originate
+    - rm -rf /home/ubuntu/.go_workspace/src/github.com/Originate/gt
```

Comment:
I have to delete this folder, since the `cp` below doesn't override all files for some strange reason.